packages/testlab/src/validate-api-spec.ts

```diff
@@ -3,20 +3,15 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import * as SwaggerParser from 'swagger-parser';
-import {OpenApiSpec} from '@loopback/openapi-spec';
+import {OpenApiSpec} from '@loopback/openapi-spec-types';
+const validator = require('swagger2openapi/validate.js');
```

Comment:
Is the TypeScript compiler able to infer the type of `validator`, or is it falling back to default `any`? It would be great to have an explicit type specified for this function, even if it means we have to write it ourselves. 

Actually, I care most about the type of `promisifiedValidator`, so if we are going go write the type def ourselves, then it's enough to describe `promisifiedValidator`.

Reply:
@bajtos hmm, could you elaborate more about
> Is the TypeScript compiler able to infer the type of validator
? 
Would `Function` the expected type or you also expect the TypeScript compiler to know its input&return types?

Reply:
> Would Function the expected type or you also expect the TypeScript compiler to know its input&return types?

Yes, I'd like the typescript compiler to know the type of input arguments and the type of the returned value.